frontend/src/App.jsx

```diff
@@ -17,6 +17,7 @@ function App() {
         <Header />
         <main>
           <Routes>
+            <Route path="*" element={<Home />} />
```

Comment:
Hmm, I have my doubt about this.
That's the syntax I'd use for a 404 error, but it might work as intended